test/services/active_challenges_cache.js

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+import ActiveChallengesCache from '../../src/services/active_challenges_cache';
+import {expect} from 'chai';
+
+describe('Active Challenges Cache', () => {
+  let activeChallengesCache;
+  const exampleChallenge = {
+    challengeId: '0xc0ffee',
+    sheltererId: '0xbeef',
+    bundleId: '0xdeadbeef',
+    count: 12,
+    blockNumber: 1,
+    logIndex: 0
+  };
+  const anotherChallengeId = {challengeId: 'anotherChallengeId'};
+
+  beforeEach(() => {
+    activeChallengesCache = new ActiveChallengesCache();
+  });
+
+  function aChallenge() {
+    return exampleChallenge;
+  }
+
+  function aChallengeWith(params) {
```

Comment:
you can merge `aChallenge` and `aChallengeWith` methods by using a params parameters with a default of {};